e2e-tests/playwright/e2e/plugins/github-insights/github-insights.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { expect, test } from "@playwright/test";
+import { UIhelper } from "../../../utils/ui-helper";
+import { Common } from "../../../utils/common";
+import { UI_HELPER_ELEMENTS } from "../../../support/pageObjects/global-obj";
+
+// Test to verify if the GitHub Insights plugin is installed - runs independently
+test("Verify GitHub Insights plugin is installed", async ({ page }) => {
+  const common = new Common(page);
+  await common.loginAsGuest();
+
+  const uiHelper = new UIhelper(page);
+
+  // Navigate to the Administration in the sidebar
+  await uiHelper.openSidebarButton("Administration");
+
+  // Click on Extensions
+  await uiHelper.openSidebar("Extensions");
+
+  // Wait for the page to load
+  await page.waitForLoadState("networkidle");
```

Comment:
Using page.waitForLoadState('networkidle') and page.waitForLoadState('domcontentloaded') is not recommended for modern web applications. These methods rely on global states that can be unreliable in dynamic applications with ongoing network requests (e.g., WebSockets or API polling). Instead, it’s better to wait for specific elements or components that indicate the page is fully loaded and interactive. For example, use
page.waitForSelector('[data-testid="something"]') to ensure tests are more reliable and maintainable.